com.unity.multiplayer.mlapi/Runtime/Core/NetworkingManager.cs

```diff
@@ -690,8 +643,8 @@ private void OnDestroy()
         {
             if (Singleton != null && Singleton == this)
             {
-                Singleton = null;
                 Shutdown();
+                Singleton = null;
```

Comment:
this was a bug causing other components like `RpcQueue` stuff to throw `NullReferenceException` at runtime due to access to `NetworkManager.Singleton` property.